### ISSUE TITLE
Solve BUG:AttributeError: module transformers has no attribute LLaMATokenizer

### ISSUE THREAD

**Issue by XuyaoWang** [tatsu-lab/stanford_alpaca#64]
I wang to follow the guide below.
> Given Hugging Face hasn't officially supported the LLaMA models, we fine-tuned LLaMA with Hugging Face's transformers library by installing it from a particular fork (i.e. this [PR](https://github.com/huggingface/transformers/pull/21955) to be merged). The hash of the specific commit we installed was 68d640f7c368bcaaaecfc678f11908ebbd3d6176.
but while click  [PR](https://github.com/huggingface/transformers/pull/21955) and try to run the example:
`
tokenizer = transformers.LLaMATokenizer.from_pretrained("/output/path/tokenizer/")
model = transformers.LLaMAForCausalLM.from_pretrained("/output/path/llama-7b/")
batch = tokenizer(
    "The primary use of LLaMA is research on large language models, including",
    return_tensors="pt", 
    add_special_tokens=False
)
batch = {k: v.cuda() for k, v in batch.items()}
generated = model.generate(batch["input_ids"], max_length=100)
print(tokenizer.decode(generated[0]))
`
I got an error:
> AttributeError: module transformers has no attribute LLaMATokenizer 
if you meet same bug, you just change  your code to:
`
tokenizer = transformers.LlamaTokenizer.from_pretrained("/output/path/tokenizer/")
model = transformers.LlamaForCausalLM.from_pretrained("/output/path/llama-7b/")
batch = tokenizer(
    "The primary use of LLaMA is research on large language models, including",
    return_tensors="pt", 
    add_special_tokens=False
)
batch = {k: v.cuda() for k, v in batch.items()}
generated = model.generate(batch["input_ids"], max_length=100)
print(tokenizer.decode(generated[0]))
`
This bug is caused by incorrect letter capitalization

**Comment by teknium1** [comment]
I see the same but fixing the capitalization didnt fix for me
![image](https://user-images.githubusercontent.com/127238744/225819984-f3a3f682-3370-4aba-b224-a383f67211f8.png)

**Comment by teknium1** [comment]
Am using transformers 4.27.1, is it a different version?

**Comment by kiran1501** [comment]
even i got the same error , please suggest how to fix this

**Comment by garcesmarc** [comment]
68d640f7c368bcaaaecfc678f11908ebbd3d6176

**Comment by bkvijay** [comment]
we are getting this error, would appreciate you help

`ValueError: Tokenizer class LlamaTokenizer does not exist or is not currently imported.`

**Comment by XuyaoWang** [comment]
> 

I didn't install transformers in pip, I download transformers in github in branch "llama_push" an move the downloaded lib file into conda

**Comment by wang1504626** [comment]
> we are getting this error, and would appreciate your help
> 
> `ValueError: Tokenizer class LlamaTokenizer does not exist or is not currently imported.`

Install the env using this:

> pip install git+https://github.com/zphang/transformers.git@68d640f7c368bcaaaecfc678f11908ebbd3d6176

**Comment by wang1504626** [comment]
Similar with previous answers, the following steps works for me:
1. Git clone from branch [llama_push](https://github.com/zphang/transformers.git). 
2. cd into this repo, git checkout 68d640f7c368bcaaaecfc678f11908ebbd3d6176
3. Run "python setup.py install" to install tranformers package.
4. Move all output files of two sub folders from this [PR](https://github.com/huggingface/transformers/pull/21955) into one single folder.

**Comment by teknium1** [comment]
Yeah you have to install from Transformers github. I had thought since it was merged it was in an updated pip package but its not yet. 
`pip install git+https://github.com/huggingface/transformers.git` works for me

**Comment by ruian0** [comment]
`LlamaTokenizer` instead of `LLaMATokenizer`

**Comment by Alro10** [comment]
Another nice solution:
Install transformer lib by running this command: `pip install -q git+https://github.com/zphang/transformers@c3dc391` that worked well here

**Comment by sunyoubo** [comment]
transformers.LLaMATokenizer  is change to transformers.LlamaTokenizer

**Comment by AkshayVerma26** [comment]
I used ```pip install git+https://github.com/huggingface/transformers.git``` for installing transformers as discussed above.
Then used the ```from transformers import LlamaForCausalLM, LlamaTokenizer```.
Still got the error: ```ImportError: cannot import name 'LlamaForCausalLM' from 'transformers'```

Please help me with this
